generics/services/aws.js

```diff
@@ -86,8 +86,70 @@ let getDownloadableUrl = function( filePath,bucketName ) {
     })
 }
 
+  /**
+     * Get aws s3 cloud signed url.
+     * @method
+     * @name getS3SignedUrl
+     * @param {String} [folderPath = ""] - link to the folder path.
+     * @param {Array} [fileName = ""] - fileName.
+     * @param {Array} bucketName - name of the bucket.  
+     * @returns {Object} - signed url and s3 file name. 
+     */
+
+let signedUrl = (folderPath = "", fileName = "",bucketName) =>{
+    return new Promise(async (resolve, reject) => {
+        try {
+            
+            if(folderPath == "" || fileName == "") {
+                throw new Error(httpStatusCode.bad_request.status);
+            }
+
+            let noOfMinutes = 30;
+            let expiry = 60 * noOfMinutes;
```

Comment:
@aman-tunerlabs - Can this expiry no of minutes be a common constant across services